packages/test-helpers/utils.js

```diff
@@ -0,0 +1 @@
+sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
https://github.com/meteor/meteor/blob/549509fd9c0abfe6c1ca42c23e6c0a0d453ad52a/packages/meteor/async_helpers.js#L170-L173

There is already a helper used in the entire project and exposed publicly to sleep.

Reply:
I've replaced all usages